src/lib/graphqlErrorHandler.js

```diff
@@ -20,6 +20,13 @@ export const shouldReportError = originalError => {
   return true
 }
 
+export const formatGravityError = error => {
+  const errorSplit = error.message.split(" - ")
+  const errorObject =
+    errorSplit && errorSplit.length > 1 ? JSON.parse(errorSplit[1]) : null
+  return errorObject
+}
```

Comment:
This module is really about reporting GraphQL errors, I’m thinking this new code could go into a Gravity errors module, including the below Gravity error type.